jscomp/others/js_bigint.res

```diff
@@ -1,3 +1,74 @@
 /*** JavaScript BigInt API */
 
-type t
+@val
+/**
+Parses the given `string` into a `bigint` using JavaScript semantics. Return the
+number as a `bigint` if successfully parsed, `null`, `undefined`, `_NaN` otherwise.
```

Comment:
not sure what `null`, `undefined`, `_NaN` refers to.

Reply:
It is invalid comment, I'll remove this part.